src/functions-templates/go/scheduled-function/netlify.toml

```diff
@@ -0,0 +1,12 @@
+[build]
+command = "make build"
+functions = "functions"
+
+[build.environment]
+GO_VERSION = "1.17.6"
```

Comment:
how will we update this? wouldn't it make more sense to use `1.17` so we always get the latest patch version? (not sure if our build system supports that)